src/repos/FlyerRepo.ts

```diff
@@ -23,14 +23,44 @@ function isFlyerFiltered(flyer: Flyer) {
 
 const getAllFlyers = async (offset = DEFAULT_OFFSET, limit = DEFAULT_LIMIT): Promise<Flyer[]> => {
   return FlyerModel.find({})
-    .sort({ date: 'desc' })
+    .sort({ startDate: 'desc' })
     .skip(offset)
     .limit(limit)
     .then((flyers) => {
       return flyers.filter((flyer) => !isFlyerFiltered(flyer));
     });
 };
 
+const getFlyersAfterDate = async (since: string, limit = DEFAULT_LIMIT): Promise<Flyer[]> => {
+  return (
+    FlyerModel.find({
+      // Get all Flyers after or on the desired date
+      endDate: { $gte: new Date(since) },
+    })
+      // Sort dates in order of most recent to least
+      .sort({ endDate: 'desc' })
+      .limit(limit)
+      .then((flyers) => {
+        return flyers.filter((flyer) => flyer !== null && !isFlyerFiltered(flyer));
+      })
+  );
+};
```

Comment:
It's generally a bad idea to mix `async` and `.then` syntax in JS/TS. Either change this to use an `await` or remove the `async` at the start of the function definition.